includes/class-edd-cache-helper.php

```diff
@@ -133,22 +121,4 @@ public function notices() {
 		}
 
 	}
-
-	/**
-	 * Prevents W3TC from adding to the cache prior to modifying data.
-	 *
-	 * @since 1.7
-	 */
-	public function w3tc_suspend_cache_addition_pre() {
```

Comment:
I'm wondering if it would be safer to leave these in, and possibly just empty them, since they're public functions--it seems like someone could be using them in custom code.

Reply:
@robincornett A search on all of GitHub for the function names results in ONLY EDD Forks.

I'm considering that safe to remove.